desc/perturbations.py

```diff
@@ -58,13 +58,13 @@ def get_deltas(things1, things2):
                 if hasattr(t1, "change_resolution") and hasattr(t2, "basis"):
                     t1.change_resolution(t2.basis.L)
                 if not np.allclose(t2.params, t1.params):
-                    deltas["d" + key[0]] = t2.params - t1.params
+                    deltas[key[0] + "_l"] = t2.params - t1.params
```

Comment:
This works but might not generalize well (if we add another profile that starts with the letter "i", "p", or "c").  I'm not sure what a better method would be, but we should at least make a comment in the code like `# FIXME: generalize`

Reply:
added a note about this. We should try to come up with a more general way to map between profile names and the names of their parameters